### DISCUSSION TITLE
https://reactnativedev.ru/rn/typescript/

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/reactnativedev.ru#2] [Announcements]
# https://reactnativedev.ru/rn/typescript/

Новые проекты, созданные с помощью React Native CLI или популярных шаблонов, таких как Ignite, по умолчанию будут использовать TypeScript

https://reactnativedev.ru/rn/typescript/

**Comment by Aleksandra-ui** [comment]
полезная статейка

**Comment by evghenimelnic** [comment]
Хороший сайт! Благодарность автору!
